extensions/wikia/ARecoveryEngine/ARecoveryUnlockCSS.class.php

```diff
@@ -5,7 +5,7 @@ class ARecoveryUnlockCSS {
 	const API_ENDPOINT = 'sp_create_csurl';
 	const CACHE_TTL = 3600 * 10; //10h
 	const TIMEOUT = 10;
-	const WIKIA_PROXY_ENDPOINT = '/__are';
+	const WIKIA_PROXY_ENDPOINT = '/__bre';
```

Comment:
I agree and mostly because of [removing the unlock CSS](https://wikia-inc.atlassian.net/browse/ADEN-3867) I'd leave it for now and I'm pushing to release recovery again :)